innkeeper/client.go

```diff
@@ -235,7 +233,7 @@ func convertRoute(id string, d *routeData, preRouteFilters, postRouteFilters []*
 }
 
 // Converts a set of Innkeeper route definitions to their eskip representation.
-func convertData(data []*routeData, preRouteFilters, postRouteFilters []*eskip.Filter) ([]*eskip.Route, []string, string) {
+func convertJsonToEskip(data []*routeData, preRouteFilters, postRouteFilters []*eskip.Filter) ([]*eskip.Route, []string, string) {
```

Comment:
question: do we still need the pre/post-route filters? Now innkeeper, with the new interface, could full control this matter.